plugins/aws_ec2/help.md

```diff
@@ -427,6 +443,7 @@ This plugin does not contain any troubleshooting information.
 
 # Version History
 
+* 1.2.0 - Connection level and action level AWS assume role support
```

Comment:
Going by recent comments is it ok to update this to be more descriptive? Maybe something like...

`Updated Connect and Action inputs to include Role ARN and External ID to provide support for AWS Assume Role functionality at both an Action and a Connection level.`